test.yml

```diff
@@ -0,0 +1,104 @@
+config:
+  target: 'http://localhost:2002/actors'
+  phases:
+    - duration: 60
+      arrivalRate: 10
+      rampTo: 50
+    - duration: 60
+      arrivalRate: 50
+      name: "Sustained max load"
+  defaults:
+    headers:
+      # x-my-service-auth: '987401838271002188298567'
+  payload:
+    path: "ArtilleryId50.csv"
+    fields:
+      - "id"
+    skipheader: true
+  http:
+    pool: 20
+scenarios:
+  - name: "GET requests"
+    # weight: 19
+    flow:
+      - loop:
+        - get:
+            url: "/{{ id }}"
+        - think: 5
+        count: 40
+  # - loop:
+  #   weight: 1
+  #   flow:
+  #     - post:
+  #         url: "/add"
```

Comment:
Learn app requires testing of both GET and POST so remember to test POST requests as well 👍